app/Http/Controllers/UserController.php

```diff
@@ -115,34 +136,33 @@ public function provideNewPassword($id)
 
     public function showListUserView()
     {
-        $users = User::where('role_id', config('const.user'))->orderBy('id', 'desc')->get();
+        $users = $this->userRepo->showListUser();
 
         return view('admin.user.userlist', compact('users'));
     }
 
     public function showChangePasswordView()
     {
-        $categories = Category::all();
+        $categories = $this->categoryRepo->getAll();
 
         return view('user.changepassword', compact('categories'));
     }
 
     public function changePassword(ChangePaswordRequest $request)
     {
-        $id = Auth::id();
-        $user = User::find($id);
-        $user->password = password_hash($request->password, PASSWORD_BCRYPT);
-        $user->save();
+        $data = [];
+        $data['password'] = password_hash($request->password, PASSWORD_BCRYPT);
+        $this->userRepo->update(Auth::id(), $data);
         $request->session()->flash('message', __('changePassword'));
 
         return redirect()->back();
     }
 
     public function changePasswordUser(ChangePasswordUserRequest $request)
     {
-        $user = User::find($request->id);
-        $user->password = password_hash($request->password, PASSWORD_BCRYPT);
-        $user->save();
+        $data = [];
+        $data['password'] = password_hash($request->password, PASSWORD_BCRYPT);
```

Comment:
```suggestion
        $data['password'] = Hash::make($request->password);
```
Dùng hàm của Laravel cho nó chuyên nghiệp nha :D 
https://laravel.com/docs/8.x/hashing